client/modules/wj-css/src/abstracts/_media.scss

```diff
@@ -1,5 +1,5 @@
 // Import `include-media` package from NPM
-@import "include-media";
+@import "../../node_modules/include-media/dist/include-media";
```

Comment:
I had really weird resolution issues when trying to import `_abstracts.scss` inside of a Svelte component, so for now all of this pathing is just done manually, unfortunately.

Reply:
I would port this comment to the SCSS file itself, so future readers of the code understand why this is path is direct